project-scratch-def.json/project-scratch-def.schema.json

```diff
@@ -1,6 +1,5 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://schemas.salesforce.com/project-scratch-def.json",
```

Comment:
We need to keep this, it's the unique identifier for the schema

Reply:
Here's a doc on it https://json-schema.org/understanding-json-schema/structuring.html#id